File: COEN/coen171/Homework0/Fibonacci/fib.java
/* Print the 10th Fibonacci number. */

class fib {
    static int fib(int n) {
	return n == 0 || n == 1 ? 1 : fib(n - 1) + fib(n - 2);
    }

    public static void main(String[] args) {
	System.out.println(fib(10));
    }
}

File: COEN/coen171/Homework0/Fibonacci/fib.c
/* Print the 10th Fibonacci number */

# include <stdio.h>

int fib(int n) {
    return n == 0 || n == 1 ? 1 : fib(n - 1) + fib(n - 2);
}

int main(void) {
    printf("%d\n", fib(10));
}
